site2/docs/tiered-storage-filesystem.md

```diff
@@ -86,7 +86,8 @@ You can configure the filesystem offloader driver in the `broker.conf` or `stand
     |---|---|---
     `managedLedgerOffloadDriver` | Offloader driver name, which is case-insensitive. | filesystem
     `fileSystemURI` | Connection address, which is the URI to access the default Hadoop distributed file system. | hdfs://127.0.0.1:9000
-    `offloadersDirectory` | Hadoop profile path. The configuration file is stored in the Hadoop profile path. It contains various settings for Hadoop performance tuning. | ../conf/filesystem_offload_core_site.xml
```

Comment:
I think we should also keep this configuration? By default, all offloaders are keeping in `offloaders` directory, but it also can configure in different directory

Reply:
The default `offloadersDirectory` is `./offloaders`, which has been mentioned on above deploy part.
```
3. Copy the Pulsar offloaders as `offloaders` in the Pulsar directory.
    ```
    mv apache-pulsar-offloaders-2.5.1/offloaders apache-pulsar-2.5.1/offloaders
    ls offloaders
    ```
```

Reply:
Other offloaders like s3/GCS docs mentioned the `offloadersDirectory` https://pulsar.apache.org/docs/en/tiered-storage-aws/#configure-aws-s3-offloader-driver. I think it will be better to keep consistent.

Reply:
So could you please keep the original field and fix its description?

Reply:
i can add the `offloadersDirectory` config back, so there will be configs below on that docs:
 * managedLedgerOffloadDriver
 * fileSystemURI
 * offloadersDirectory
 * fileSystemProfilePath

Reply:
fix in https://github.com/apache/pulsar/pull/13397/commits/b8fc1a79a75790c570a54a35c7e04cb4b79e0049